### ISSUE TITLE
focus_follows_pointer should not raise windows

### ISSUE THREAD

**Issue by Stebalien** [baskerville/bspwm#41]
Currently, `focus_follows_pointer` raises windows on focus. This makes working with floating windows (dialogs etc) extremely frustrating. As far as I can tell, it should be possible to give a window keyboard focus without raising it but I don't know nearly enough about xcb to implement this myself.
